### ISSUE TITLE
Won't build in Node 6.0.0

### ISSUE THREAD

**Issue by jbinto** [frig-js/frigging-bootstrap#36]
When running `npm run preversion`, the minified build fails on Node 6.0.0

```
ERROR in ../frigging-bootstrap/~/css-loader!../frigging-bootstrap/~/stylus-loader!../frigging-bootstrap/src/stylesheets/index.styl
Module build failed: TypeError: /Users/jbinto/dev/vendor/frigging-bootstrap/src/stylesheets/index.styl:7:1
   3| @import "./components/switch/styles.styl"
   4| @import "./components/timepicker.styl"
   5| @import "./components/colorpicker.styl"
   6| @import "./components/saved.styl"
   7|
------^

Path must be a string. Received undefined

    at assertPath (path.js:7:11)
    at extname (path.js:1429:5)
    at new SourceMapper (/Users/jbinto/dev/vendor/frigging-bootstrap/node_modules/stylus/lib/visitor/sourcemapper.js:41:7)
    at Renderer.render (/Users/jbinto/dev/vendor/frigging-bootstrap/node_modules/stylus/lib/renderer.js:94:9)
    at /Users/jbinto/dev/vendor/frigging-bootstrap/node_modules/stylus-loader/index.js:149:12
    at tryCatchReject (/Users/jbinto/dev/vendor/frigging-bootstrap/node_modules/when/lib/makePromise.js:840:30)
    at runContinuation1 (/Users/jbinto/dev/vendor/frigging-bootstrap/node_modules/when/lib/makePromise.js:799:4)
    at Fulfilled.when (/Users/jbinto/dev/vendor/frigging-bootstrap/node_modules/when/lib/makePromise.js:590:4)
    at Pending.run (/Users/jbinto/dev/vendor/frigging-bootstrap/node_modules/when/lib/makePromise.js:481:13)
    at Scheduler._drain (/Users/jbinto/dev/vendor/frigging-bootstrap/node_modules/when/lib/Scheduler.js:62:19)
 @ ../frigging-bootstrap/src/stylesheets/index.styl 4:14-124
```

**Comment by jbinto** [comment]
Lots of packages seem to have had problems initially with Node 6, likely if we just upgrade some dependencies this will fix itself. Hint: `stylus`, `stylus-loader`, `css-loader`, `webpack` itself...

**Comment by jbinto** [comment]
Fixed in #43.
